include/yaml-cpp/node/detail/impl.h

```diff
@@ -56,6 +56,39 @@ struct get_idx<Key, typename std::enable_if<std::is_signed<Key>::value>::type> {
   }
 };
 
+template <typename Key, typename Enable = void>
+struct remove_idx {
+  static char remove(std::vector<node*>& /* sequence */, const Key& /* key */) {
```

Comment:
Why does this return a 'char'? It looks like in all implementations, it either returns 0 or 1.